Resources/CDMapPatches/box.yml

```diff
@@ -0,0 +1,4 @@
+- !type:CDSpawnEntityMapPatch
+  worldPosition: -5.999672,-16.99885
+  id: LockerLostAndFound
+...
```

Comment:
You should delete these `...` lines. They are an artifact of the YAML library SS14 uses.

Reply:
Should be fixed now.